docs/src/utils/exampleContexts.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Get the Webpack Context for doc site example groups.
```

Comment:
nit: suggest to omit 'Get' verb as these exports just provide corresponding context modules, while 'get' action happens on the client side. 

Also, while 'groups' is also correct, would suggest to rename this context to `exampleIndexContext` - it will be easier for the reader to immediately infer the files implied by this context, without any additional necessity to remind terminology that is used by docs.

Reply:
Agree, renamed to `exampleIndexContext` and removed the `get` verb 👍